CHANGES.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## 1.11.16 (core) / 0.27.16 (libraries)
+
+### New
+
+- Proxy server heartbeat TTL is now configurable behind `DAGSTER_GRPC_PROXY_HEARTBEAT_TTL_SECONDS` (default 30 seconds).
```

Comment:
nit: this is a bit opaque, maybe:

"The proxy GRPC server heartbeat TTL can now be configured with the `DAGSTER_GRPC_PROXY_HEARTBEAT_TTL_SECONDS` env var (default remains 30 seconds).